app/models/grda_warehouse/client_notes/base.rb

```diff
@@ -40,6 +40,14 @@ class Base < GrdaWarehouseBase
       where(type: types)
     end
 
+    scope :active, -> do
+      where.not(id: expired)
+    end
+
+    scope :expired, -> do
+      where('expiration_date <= ?', Date.current)
```

Comment:
ActiveRecord can do this natively with:
```suggestion
      where(expiration_date: .. Date.current)
```

Reply:
cleaned up [here](https://github.com/greenriver/hmis-warehouse/pull/5160/commits/7cce5e5a24f96e79d8341e700963bd0b8a6ff59f)